resources/icons/16px/shield.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49.2 (51160) - http://www.bohemiancoding.com/sketch -->
```

Comment:
🤔 this markup looks quite different from the other icons... seems the others were generated with Illustrator but this one with Sketch. not a problem, just noting an observation.

Reply:
yeah, the markup differs between these two. Currently we have a mix of Sketch and AI exported icons.